internal/output/plain.go

```diff
@@ -263,6 +264,10 @@ func sprintTable(slice []interface{}) (string, error) {
 				stringValue = trimValue(stringValue, termWidth)
 			}
 
+			if funk.Contains(field.opts, string(EmptyNil)) {
+				stringValue = useEmptyNil(stringValue)
```

Comment:
```suggestion
			if funk.Contains(field.opts, string(EmptyNil)) && stringValue == nilText {
				stringValue = ""
```

And delete `useEmptyNil`